beta/experience-platform-extension/tutorials/tutorial-5-analytics-edge-extension.md

```diff
@@ -0,0 +1,93 @@
+# Using Analytics edge extension 
+
+{% hint style="warning" %}
+The Adobe Experience Platform Edge mobile extension is currently in BETA. Use of this extension is by invitation only. Please contact your Adobe Customer Success Manager to learn more and get access to the materials for this tutorial.
+{% endhint %}
+
+## Prerequisites for this tutorial
+
+* Access to Adobe Experience Platform
+* Minimal Swift / Android development knowledge 
+* Knowledge about the AEP Edge extension
+* Completion of [Assignment 1 - AEP Edge extension setup and XDM usage](https://aep-sdks.gitbook.io/docs/beta/experience-platform-extension/tutorials/tutorial-1-edge-extension-setup) (required to have a mobile property in Launch with Edge configuration setup).
+
+### Enable Analytics in Edge configuration
+
+In Adobe Experience Platform Launch, navigate to your mobile property and select Edge Configurations from the left panel, then select the configuration created in Assignment 1
+
+* Select development environment, enable `Adobe Analytics` and add a development report suite. If you don't have one, create a new report suite for this exercise using the Mobile template.
+
+   ![](../../../.gitbook/assets/edge_analytics_config.png)
+
+
+### Download the sample application
+
+{% tabs %}
+
+{% tab title="iOS" %}
+
+#### Swift
+
+Download the iOS Swift Sample application from [https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-5](https://github.com/adobe/aepsdk-sample-app-ios/archive/beta-assignment-5.zip).
+
+To get started, follow the steps described in [AEP SDK Sample App Swift - Installation](https://github.com/adobe/aepsdk-sample-app-ios/tree/beta-assignment-5#installation).
+{% endtab %}
+{% endtabs %}
+
+### Set up the required fields
+
```

Comment:
This part can be done after Install the AnalyticEdge Extension section.  But before Init Extension.